### ISSUE TITLE
See All People links to wrong category

### ISSUE THREAD

**Issue by eidolonnight** [WordPress/wporg-news-2021#430]
The "[See All People](https://wordpress.org/news/category/community/)" link in WordPress/news currently links to the [Community category](https://wordpress.org/news/category/community/), which is a broad category for community content. This link should instead link to the [Interviews category](https://wordpress.org/news/category/interviews/) that houses all People of WordPress interviews. The Interviews category should also use the layout of the Community category.

![Image](https://github.com/user-attachments/assets/2aa206eb-61b8-47eb-bcac-5eeb05191aca)

![Image](https://github.com/user-attachments/assets/49493a0f-7db1-477b-8839-ac8b7955da24)

**Comment by ryelle** [comment]
@eidolonnight Just for clarity — this is the expected behavior: "See all people" should link to Community (#266) and the "Community" archive (including non-people posts) is meant to be styled like this (#148).

**Comment by eidolonnight** [comment]
Sorry for the hasty issue @ryelle . Here is the expected behavior:

- "See all people" should link to [Interviews category](https://wordpress.org/news/category/interviews/) - This category only has the People of WordPress posts.
- [Interviews category](https://wordpress.org/news/category/interviews/) should inherit the same style as the [Community category](https://wordpress.org/news/category/community/).

As an aside, I assume the Interviews category is what is used to show the featured images in the "People of WordPress" section on the main news page? The extraneous community posts don't appear there, only faces.

**Comment by ryelle** [comment]
Okay, I just wanted to clarify that you want to change the existing behavior to your new proposed outcome, not that something broke and there's a bug.

> [Interviews category](https://wordpress.org/news/category/interviews/) should inherit the same style as the [Community category](https://wordpress.org/news/category/community/).

Can "Community" keep the current style too, or do they need to swap styles?

> I assume the Interviews category is what is used to show the featured images in the "People of WordPress" section on the main news page? 

No, this uses the "[People of WordPress](https://wordpress.org/news/tag/people-of-wordpress/)" tag (tag archives are not used on the site, so it's not styled).

For prioritization, do you need this change by a certain date?

**Comment by eidolonnight** [comment]
> I just wanted to clarify that you want to change the existing behavior to your new proposed outcome, not that something broke and there's a bug.

Correct. This is not a technical bug, though clicking a "See All People" link and not seeing people breaks the user experience.

> Can "Community" keep the current style too, or do they need to swap styles?

Yes, "Community" can keep the current style too. I see no reason to change it.

> No, this uses the "[People of WordPress](https://wordpress.org/news/tag/people-of-wordpress/)" tag (tag archives are not used on the site, so it's not styled).

I see. Thanks for the clarification.

> For prioritization, do you need this change by a certain date?

We do sometimes share these stories as we approach State of the Word, so before then would be great. That said, aside from traffic sent here, these stories are not the most popular. I'm unsure of the technical lift, so the prioritization is ultimately up to you.
